### ISSUE TITLE
Uploading files

### ISSUE THREAD

**Issue by rama41222** [jeremydaly/lambda-api#157]
### How to I handle files / multipart form data using lambda-api ? 

* I tried using multer and multer-s3 but it didn't work, I got `req.pipe not found error`

**Comment by sebastianDejoy** [comment]
yes please, busboy is limited...multer rules on NodeJS. Please add support to that.

**Comment by sebastianDejoy** [comment]
Any insight about piping on this framework?

**Comment by alejuanito** [comment]
Hi, were you able to process a multipart / form-data file? is it possible to do it with this library?

**Comment by sebastianDejoy** [comment]
Hey @alejuanito, yeap i was. The best approach is to use `presigned` urls that have a reasonable payload limit (50GB) as opposed to lambda/APIGW that is around 6MB.  

I enabled an endpoint that return a presigned url:
```
const { getSignedUrl } = require("@aws-sdk/s3-request-presigner");
const { S3Client, GetObjectCommand } = require("@aws-sdk/client-s3");

const client = new S3Client(clientParams);
const command = new GetObjectCommand(getObjectParams);
const url = await getSignedUrl(client, command, { expiresIn: 3600 }); // This url expires in 15 minutes
return url; //Return this url as a response in your lambda handler.
```

Then your API consumers (mobile aps) can upload whatever file they want through that presigned url: 
`curl -v --upload-file ${fileName} ${presigned_url}`

Make sure to grant access from your S3 bucket.
